### ISSUE TITLE
Navigator Error

### ISSUE THREAD

**Issue by paulsimvient** [ContinuumIO/anaconda-issues#7124]
## Main error
Application <b>spyder</b> launch may have produced errors.
## Traceback
```
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0
[warn] kq_init: detected broken kqueue; not using.: Undefined error: 0

```
## System information
```
platform: osx-64
version: 1.6.2
conda: 4.3.30
qt: 5.6.2
language: en
python: 2.7.13
os: Darwin;16.6.0;Darwin Kernel Version 16.6.0: Fri Apr 14 16:21:16 PDT 2017; root:xnu-3789.60.24~6/RELEASE_X86_64;x86_64;i386
pyqt: 5.6.0
```

**Comment by goanpeca** [comment]
**See Issue #1778 for more information on how to fix this.**

---

Closing as duplicate of #1778


---

Please remember to update to the latest version of Navigator to include
the latest fixes.

Open a terminal (on Linux or Mac) or the Anaconda Command Prompt (on windows)
and type:

```
$ conda update anaconda-navigator
$ conda update navigator-updater
```

**Comment by paulsimvient** [comment]
Beautiful thank you!!

Paul Cummings | Senior Fellow | +1.571.337.7412 main | paul.cummings@icf.com | icf.com
ICF | 9300 Lee Highway, Fairfax VA 22031 USA




On Sun, Nov 5, 2017 at 12:37 PM -0500, "Gonzalo Pe?a-Castellanos" <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #7124<https://github.com/ContinuumIO/anaconda-issues/issues/7124>.

-
